ocs_ci/ocs/ui/page_objects/fusion_access_ui.py

```
@@ -0,0 +1,248 @@
+        # Open the dropdown
+        self.base_ui.do_click(FDF_SAN_LOCATORS["secret_key_dropdown"])
+        logger.info("Opened Secret key dropdown")
+
+        # Wait for dropdown options to appear in the DOM (up to 15 seconds)
+        options_xpath = "//ul[contains(@class,'pf-v6-c-menu__list')]//li//button"
+        logger.info("Waiting for Secret key dropdown options to appear...")
+        try:
+            WebDriverWait(self.base_ui.driver, 15).until(
+                EC.presence_of_element_located((By.XPATH, options_xpath))
+            )
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)